libfwupdplugin/fu-volume.c

```diff
@@ -843,6 +868,7 @@ fu_volume_new_by_kind(const gchar *kind, GError **error)
 	devices = fu_common_get_block_devices(error);
 	if (devices == NULL)
 		return NULL;
+	g_info("Looking for volumes of type %s", kind);
```

Comment:
g_debug here -- it's not something we want in syslog really

Reply:
Remember `g_debug` == `-vv` and `g_info` = `-v`

Reply:
I'm thinking of g_log maybe. Too much relaxing on PTO. 🙂